Makefile

```
@@ -35,6 +35,7 @@ dist: all
 	docker save $(IMAGE_NAME) | gzip > $(DISTFILE)
 
 # Build the container image.
+build: export DOCKER_BUILDKIT = 1
 build:
 	docker build $(DOCKER_BUILD_FLAGS) \
```

```suggestion
build:
	DOCKER_BUILDKIT=1 docker build $(DOCKER_BUILD_FLAGS) \
```

This looks a bit cleaner to me since we don't need to export `DOCKER_BUILDKIT` for all of the commands in the `build` target (not that there are any others right now...).  Not a strong preference though.

---

I'm fine with either. The only reason I typically add to all commands is if you end up with multiple docker commands in the build target it can export to all of them at once instead of adding to each.